app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -728,17 +689,28 @@ impl Node {
             // ths user hovers the drag area. The input port manager merges this information with
             // port hover events and outputs the final hover event for any part inside of the node.
 
-            let drag_area = &model.drag_area.events_deprecated;
-            drag_area_hover <- bool(&drag_area.mouse_out,&drag_area.mouse_over);
-            model.input.set_hover  <+ drag_area_hover;
+            let background_enter = model.background.on_event::<mouse::Enter>();
+            let background_leave = model.background.on_event::<mouse::Leave>();
+            background_hover <- bool(&background_leave, &background_enter);
+            let input_enter = model.input.on_event::<mouse::Over>();
+            let input_leave = model.input.on_event::<mouse::Out>();
+            input_hover <- bool(&input_leave, &input_enter);
+            node_hover <- background_hover || input_hover;
+            node_hover <- node_hover.debounce().on_change();
+            model.input.set_hover <+ node_hover;
             model.output.set_hover <+ model.input.body_hover;
             out.hover <+ model.output.body_hover;
 
 
             // === Background Press ===
 
-            out.background_press <+ model.drag_area.events_deprecated.mouse_down_primary;
-            out.background_press <+ model.input.on_background_press;
+            let background_press = model.background.on_event::<mouse::Down>();
+            let input_press = model.input.on_event::<mouse::Down>();
+            input_as_background_press <- input_press.gate(&input.set_edit_ready_mode);
+            trace input_as_background_press;
```

Comment:
trace - also check other places before merge, as its hard to catch all of them during review